src/clients/client.rs

```diff
@@ -99,11 +99,11 @@ impl Client {
     ///
     /// Demonstrates basic usage.
     /// ```no_run
-    /// use mini_redis::client;
+    /// use mini_redis::clients;
     ///
     /// #[tokio::main]
     /// async fn main() {
-    ///     let mut client = client::connect("localhost:6379").await.unwrap();
+    ///     let mut client = clients::connect("localhost:6379").await.unwrap();
```

Comment:
Having to write `clients::connect` is a bit awkward.

Reply:
Perhaps it should be a method on the `Client` type instead?

Reply:
Agree. Maybe we can have `connect` methods on the `Client`, `BlockingClient`, and `BufferedClient` types?